databricks/koalas/base.py

```diff
@@ -136,10 +136,13 @@ def _with_new_scol(self, scol: spark.Column) -> IndexOpsMixin
         Creates new object with the new column
     """
     def __init__(self, internal: _InternalFrame, kdf):
+        from databricks.koalas.series import Series
         assert internal is not None
         assert kdf is not None and isinstance(kdf, DataFrame)
         self._internal = internal  # type: _InternalFrame
         self._kdf = kdf
+        if isinstance(self, Series):
+            self._index = kdf.index
```

Comment:
and .. remove the changes here ..

Reply:
ah, sorry for carelessness.

thanks :)